### PR TITLE
Add deprecation message to readme

### PR DESCRIPTION
**Pull request by levb**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,11 @@
-# Test app for Mattermost
+# DEPRECATED
 
-## Install
+This repopsitory is no longer used, and the app has moved to https://github.com/mattermost/mattermost-plugin-apps/tree/master/test/app
```
Comment (catalintomai) [inline README.md:3]: repopsitory  -> repository

**Review by catalintomai — APPROVED** [review]

**Comment by hanzei** [comment]
Should the repo get archived?

**Comment by levb** [comment]
@hanzei yes, I don't think I have the privs

**Comment by hanzei** [comment]
https://git.internal.mattermost.com/ops/github-management/-/merge_requests/118

**Pull request merged**